test/conformance/kuttl/generate/clusterpolicy/standard/clone/sync/cpol-clone-list-sync-update/manifests.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bootstrap-config
+  namespace: default
+  labels:
+    allowedToBeCloned:  "true"
+data:
+  initial_lives: "15"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: image-secret
+  namespace: default
+  labels:
+    allowedToBeCloned:  "true"
+type: kubernetes.io/basic-auth
+stringData:
+  username: admin
+  password: t0p-Secret-super
```

Comment:
You cannot use the `stringData` object when asserting. All Secrets are required to store data in base64-encoded format. The `stringData` object is only used for ease of creation. When the Secret resource is persisted, its data is done so in base64. This is the reason conformance tests are failing.

Reply:
that's correct, i am not comparing with `stringData` instead, with `data` field which is base64-encoded.

Reply:
tests passed.